lib/core/src/Cardano/Pool/Metrics.hs

```diff
@@ -213,8 +219,11 @@ newStakePoolLayer db@DBLayer{..} nl tr = StakePoolLayer
         when (Map.null distr || Map.null prod) $ do
             computeProgress nodeTip >>= throwE . ErrMetricsIsUnsynced
 
+
+        let (_, bp) = staticBlockchainParameters nl
```

Comment:
Neat picking but, Readability would be enhanced by moving this in a `where ` clause because it doesn't depend on the surrounding context.

Reply:
👍 Done.